test/testcil.pl

```diff
@@ -695,6 +695,7 @@ sub addToGroup {
 
 addTest("testrunc11/c11-generic");
 addTest("testrunc11/c11-caserange");
+addTest("testrunc11/c11-extendedFloat");
```

Comment:
I think this isn't part of C11 at all, but some completely separate standard called ISO/IEC TS 18661-3:2015, which seems like it will be part of C23 according to https://en.cppreference.com/w/c/experimental.

Reply:
Yes, the question is whether we want to have a separate entry in the Makefile for just this, or whether we leave it under C11 as it is at least related. (`testrunc11` basically says runs with `gcc -std=c11`). I'd opt for the latter.

Reply:
Maybe at least a comment here to mention that it's technically separate from C11.